include/wx/msw/wx_dpi_aware.manifest

```diff
@@ -5,7 +5,14 @@
       <assemblyIdentity type="win32" name="Microsoft.Windows.Common-Controls" version="6.0.0.0" processorArchitecture="*" publicKeyToken="6595b64144ccf1df" language="*" />
     </dependentAssembly>
   </dependency>
-  <trustInfo xmlns="urn:schemas-microsoft-com:asm.v3">
+
+  <!--
+  when compiling with CMake, Clang and msys2
```

Comment:
Do you have more information about this problem? I'm a bit afraid of changing this blindly, it worked without problems for many years, after all.